lib/addons/opa-gatekeeper/index.ts

```diff
@@ -0,0 +1,60 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+
+/**
+ * Configuration options for the add-on.
+ */
+export interface OpaGatekeeperAddOnProps {
+
+    /**
+     * Namespace where OPA Gatekeeper will be installed
+     * @default kube-system
+     */
+    namespace?: string;
+
+    /**
+     * Helm chart version to use to install.
+     * @default 3.6.0-beta.3
+     */
+    version?: string;
+
+    /**
+     * Values for the Helm chart.
```

Comment:
Please add a reference to the chart values supported by helm that you mentioned in slack.